scripts/setup_iso.py

```diff
@@ -37,12 +54,6 @@ def do_setup_repo(osname, osver, src, dest, link):
         print 'Found existing directory, bailing...'
         sys.exit(1)
 
-    if osname is 'ESXi':
-        shutil.copytree(src, dstpath)
-
-    if osname is 'RHEL' or osname is 'Centos':
```

Comment:
Should be correct. This piece of code is there before my code change, and I tested with CentOS iso after my change(without changing the upper or lower case of CentOS), and it worked